src/aap_eda/api/views/eda_credential.py

```diff
@@ -208,6 +209,10 @@ def partial_update(self, request, pk):
                 model_to_dict(eda_credential),
             )
 
+        # clear cache if updating analytics credentials
```

Comment:
Will the caching and clearing logic work considering they are running on difference containers?

Reply:
Your suggestions?

Reply:
It makes sense. The collection mostly happens every four hours or sometimes runs manually.

Reply:
The cache logic is removed